weTranslate/FavoriteCategoriesViewController.swift

```diff
@@ -67,6 +68,7 @@ final class FavoriteCategoriesViewController: UIViewController {
         super.viewDidLoad()
 
         view.backgroundColor = Color.brand
+        navigation()
```

Comment:
We can set the `title` here (localized too).
We should do a `NavigationController` (UINavigationContoller) and use that everywhere so we can style it once.